# HG changeset patch
# User andrew
# Date 1305128024 14400
# Node ID 6b74421f46b04f8cbe898ed4d8896b46be70becf
# Parent  fe6a8ddfe33d460fe19867c6625ee6390934e4b7
7042040: Remove disk space sanity check
Summary: Remove outdated disk space checks using df
Reviewed-by: ohair, omajid

diff -r fe6a8ddfe33d -r 6b74421f46b0 make/common/shared/Platform.gmk
--- openjdk/jdk/make/common/shared/Platform.gmk	Tue May 10 11:58:42 2011 +0200
+++ openjdk/jdk/make/common/shared/Platform.gmk	Wed May 11 11:33:44 2011 -0400
@@ -75,7 +75,6 @@
 #     USING_CYGWIN                windows only: true or false
 #     WINDOWS_NT_VERSION_STRING   windows only: long version name
 #     REQUIRED_OS_VERSION         required OS version, e.g. 5.10, 2.4
-#     REQUIRED_FREE_SPACE         minimum disk space needed for outputdir
 #     ISHIELD_TEMP_MIN            windows only: minimum disk space in temp area
 #     REQUIRED_ZIP_VER            required version of zip
 #     REQUIRED_UNZIP_VER          required version of unzip
@@ -171,13 +170,6 @@
   else
     REQUIRED_OS_VERSION = 5.8
   endif
-  # Minimum disk space needed as determined by running 'du -sk' on 
-  #    a fully built workspace.
-  ifeq ($(ARCH_FAMILY), sparc)
-    REQUIRED_FREE_SPACE=1300000
-  else
-    REQUIRED_FREE_SPACE=1040000
-  endif
   # How much RAM does this machine have:
   MB_OF_MEMORY=$(shell /etc/prtconf | fgrep 'Memory size:' | expand | cut -d' ' -f3)
 endif
@@ -247,9 +239,6 @@
 
   # Suffix for file bundles used in previous release
   BUNDLE_FILE_SUFFIX=.tar.gz
-  # Minimum disk space needed as determined by running 'du -sk' on 
-  #    a fully built workspace.
-  REQUIRED_FREE_SPACE=1460000
   LINUX_VERSION_INFO = /etc/redhat-release
   ifeq ($(ARCH_DATA_MODEL), 32)
     REQUIRED_LINUX_VER = Advanced Server
@@ -371,9 +360,6 @@
   ARCH_VM_SUBDIR=jre/bin
   # Suffix for file bundles used in previous release
   BUNDLE_FILE_SUFFIX=.tar
-  # Minimum disk space needed as determined by running 'du -sk' on 
-  #    a fully built workspace.
-  REQUIRED_FREE_SPACE=500000
   # ISHIELD_TEMP_MIN is the difference of an empty C:\TEMP vs. one after a
   #     bundles build on windows.
   ISHIELD_TEMP_MIN=250000
diff -r fe6a8ddfe33d -r 6b74421f46b0 make/common/shared/Sanity-Settings.gmk
--- openjdk/jdk/make/common/shared/Sanity-Settings.gmk	Tue May 10 11:58:42 2011 +0200
+++ openjdk/jdk/make/common/shared/Sanity-Settings.gmk	Wed May 11 11:33:44 2011 -0400
@@ -189,8 +189,6 @@
 endif
 ALL_SETTINGS+=$(call addRequiredVersionSetting,OS_VERSION)
 ALL_SETTINGS+=$(call addRequiredSetting,OS_NAME)
-ALL_SETTINGS+=$(call addRequiredSetting,TEMP_FREE_SPACE)
-ALL_SETTINGS+=$(call addRequiredSetting,FREE_SPACE)
 ALL_SETTINGS+=$(call addRequiredSetting,MB_OF_MEMORY)
 
 
diff -r fe6a8ddfe33d -r 6b74421f46b0 make/common/shared/Sanity.gmk
--- openjdk/jdk/make/common/shared/Sanity.gmk	Tue May 10 11:58:42 2011 +0200
+++ openjdk/jdk/make/common/shared/Sanity.gmk	Wed May 11 11:33:44 2011 -0400
@@ -46,8 +46,6 @@
 # Settings and rules to validate the JDK build environment.
 
 ifeq ($(PLATFORM), solaris)
-  FREE_SPACE := $(shell $(DF) -b $(OUTPUTDIR) | $(TAIL) -1 | $(NAWK) '{print $$2;}')
-  TEMP_FREE_SPACE := $(shell $(DF) -b $(TEMP_DISK) | $(TAIL) -1 | $(NAWK) '{print $$2;}')
   REQ_PATCH_LIST = $(JDK_TOPDIR)/make/PatchList.solaris
   ifeq ($(ARCH_FAMILY), sparc)
     PATCH_POSITION = $$4
@@ -57,8 +55,6 @@
 endif
 
 ifeq ($(PLATFORM), linux)
-  FREE_SPACE := $(shell $(DF) --sync -kP $(OUTPUTDIR) | $(TAIL) -1 | $(NAWK) '{print $$4;}')
-  TEMP_FREE_SPACE := $(shell $(DF) --sync -kP $(TEMP_DISK) | $(TAIL) -1 | $(NAWK) '{print $$4;}')
   ifeq ($(ARCH), amd64)
     LINUX_VERSION := $(shell \
         if [ -r "$(LINUX_VERSION_INFO)" ] ; then \
@@ -82,8 +78,6 @@
 endif
 
 ifeq ($(PLATFORM), windows)
-  FREE_SPACE := $(shell $(DF) -kP $(OUTPUTDIR) | $(TAIL) -1 | $(NAWK) '{print $$4;}')
-  TEMP_FREE_SPACE := $(shell $(DF) -kP $(TEMP_DISK) | $(TAIL) -1 | $(NAWK) '{print $$4;}')
   # Localized systeminfo has localized labels, but not localized values.
   _WINDOWS_VERSION := \
     $(shell systeminfo 2> $(DEV_NULL) | grep 'Microsoft' | grep 'Windows' | \
@@ -675,18 +669,6 @@
 	    "      Either obtain these permissions or set ALT_OUTPUTDIR. \n" \
 	    "" >> $(ERROR_FILE) ; \
 	fi
-	@#
-	@# OUTPUTDIR must have enough free space...
-	@#
-	@if [ $(FREE_SPACE) -lt $(REQUIRED_FREE_SPACE) ]; then \
-	  $(ECHO) "WARNING: You may not have enough free space in your OUTPUTDIR. The \n" \
-	    "        current value of OUTPUTDIR is \n" \
-	    "            $(OUTPUTDIR) \n" \
-	    "        You need "$(REQUIRED_FREE_SPACE)" Kbytes free on this device to build \n" \
-	    "        and it appears that only "$(FREE_SPACE)" Kbytes are free. \n" \
-	    "        Either obtain more space or set ALT_OUTPUTDIR to a larger disk. \n" \
-	    "" >> $(WARNING_FILE) ; \
-	fi
 
 ######################################################
 # if specified, ALT_BOOTDIR must point to non-relative path if set
